samza-core/src/test/java/org/apache/samza/example/PageViewCounterExample.java

```diff
@@ -42,32 +40,29 @@
 public class PageViewCounterExample implements StreamApplication {
 
   @Override public void init(StreamGraph graph, Config config) {
+    MessageStream<PageViewEvent> pageViewEvents =
+        graph.getInputStream("pageViewEventStream", (k, m) -> (PageViewEvent) m);
+    OutputStream<String, PageViewCount, PageViewCount> pageViewEventPerMemberStream = graph
+        .getOutputStream("pageViewEventPerMemberStream", m -> m.memberId, m -> m);
```

Comment:
minor: Not sure if there is like a single pattern we want to advocate. inlining versus having `OutputStream`s being pre-declared. If so, would be nice to use them consistently.